articles/node-vs-nuxt/index.md

```diff
@@ -0,0 +1,242 @@
+| What it is?         | Node.js is a server-side JavaScript runtime environment.                          | Nuxt is a framework used for building universally rendered web applications.                   |
+| Primary Use         | Server-side scripting, creating application programming interfaces.               | Making single page applications and univerally rendered applications.                          |
+| Built on            | Chrome V8 Engine                                                                  | Vue.js                                                                                         |
+| Languages Supported | JavaScript, TypeScript                                                            | JavaScript, TypeScript                                                                         |
+| Open-Sourced        | Yes                                                                               | Yes                                                                                            |
+| When should I use?  | Node.js is ideal for creating web servers and application programming interfaces. | Nuxt can be used to create performant user interfaces with server-side rendering and good SEO. |
+
+### Create a Node.js App
+
+Let us create a basic Node.js application.
+
+- To start, create a directory called hello-node.
+
+```bash
+mkdir hello-node
+
+cd hello-node
+```
+
+- Now, let us initiate a Node.js application by using the Node Package Manager.
+
+```bash
+npm init
+```
+
+![Node App Creation](/engineering-education/node-vs-nuxt/hello-node.png)
+
+- The above command initializes a project and creates a `package.json` file in the current directory. The package.json file is used to manage dependencies of the application.
+
+- Create a new file `index.js`.
+
+- For our script, let us install dependencies using the node package manager.
+
+```bash
+npm i emoji-random node-emoji
+```
+
+- Now, let us start writing our Node.js script that prints a random emoji to the console.
+
+```js
+// Importing process.stdin to take user input
+let std_input = process.stdin;
+
+// Importing packages
+let re = require("emoji-random");
+let emoji = require("node-emoji");
+
+console.log("1. print");
+console.log("2. exit");
+
+// When data is entered into stdin, this block executes
+std_input.on("data", (data) => {
+  if (data != "exit") {
+    let randEmoji = re.random();
+    var finalEmoji = emoji.get(randEmoji);
+
+    // Print to the console
+    console.log(
+      "Here is the " + randEmoji.split(":")[1] + "emoji: " + finalEmoji
+    );
+    process.exit();
+  } else {
+    console.log(data);
+    console.log("exiting!");
+    process.exit();
+  }
+});
+```
+
+- We have thus created a Node.js application.
+
+### Create a Nuxt App
+
+- To create a Nuxt application, we use the `create-nuxt-app` utility.
+
+```bash
+npx create-nuxt-app hello-nuxt
```

Comment:
You should provide more instructions here regarding the choices after this command is run. Beginners won't know if they need to choose a UI or testing framework or not.